fusee/fusee-secondary/src/kernel_patches.c

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2018 Atmosphère-NX
+ * Copyright (c) 2019 m4xw <m4x@m4xw.net>
```

Comment:
I don't object to this necessarily, but we haven't altered the copyright notices on any other files following pull requests to the project. Do you consider this important?

Reply:
That's usually what projects do to track contributions (history could be squashed at some point etc.)
Also I am not part of the Atmosphere team, if I need to be contacted for lets say a theoretical license change, my contact information will be present